edx_event_bus_kafka/internal/producer.py

```diff
@@ -36,6 +38,17 @@
 EVENT_TYPE_HEADER_KEY = "ce_type"
 
 
+def record_error(error, context):
```

Comment:
To make it more clear why this isn't used for some of the other errors, maybe:
1. Rename to record_event_producing_error?
2. Please add arguments to docstring. Especially helpful to know what context is meant to be.

Reply:
[fixed]